components/raftstore-v2/src/raft/storage.rs

```diff
@@ -41,19 +43,102 @@ pub fn write_initial_states(wb: &mut impl RaftLogBatch, region: Region) -> Resul
 }
 
 /// A storage for raft.
+#[derive(Debug)]
 pub struct Storage<ER> {
```

Comment:
Should it be PeerStorage instead as it's peer specific. Also same naming would be easier to understand and compare with v1's implementation.

Reply:
The struct is defined in raft, so it's used for raft. `PeerStorage` is not a good name in my opinion.

Reply:
> The struct is defined in raft, so it's used for raft. `PeerStorage` is not a good name in my opinion.
For its function, is it similar to PeerStorage in v1? If yes, I suggest to use the consistent name between v1 and v2 for easier code navigation.  Good/bad name is more or less about personal preference, consistent name is not.

Reply:
At least should add a comment to clarify it's v2 version of PeerStorage.  Same to other structs if there's  a 1:1 mapping with v1 if name is different. It will help the code understanding.